skll/experiments.py

```diff
@@ -932,20 +933,19 @@ def _create_learner_result_dicts(task_results,
         learner_result_dict['fold'] = 'average'
 
         if result_table:
-            result_table = PrettyTable(["Label", "Precision", "Recall",
-                                        "F-measure"],
-                                       header=True)
-            result_table.align = "r"
-            result_table.align["Label"] = "l"
-            result_table.float_format = '.3'
+            headers = ["Label", "Precision", "Recall", "F-measure"]
+            rows = []
             for actual_label in labels:
                 # Convert sums to means
                 prec_mean = prec_sum_dict[actual_label] / num_folds
                 recall_mean = recall_sum_dict[actual_label] / num_folds
                 f_mean = f_sum_dict[actual_label] / num_folds
-                result_table.add_row([actual_label] +
-                                     [prec_mean, recall_mean, f_mean])
+                rows.append([actual_label] + [prec_mean, recall_mean, f_mean])
 
+            result_table = tabulate(rows, 
+                                    headers=headers, 
+                                    floatfmt=".3f", 
+                                    tablefmt="grid",)
```

Comment:
- Again. please remove the extra whitespace after the commas.
- Change `grid` to `psql`  since that format better matches the last precision/recall table in the results file.